flow-nodes/api-builder-plugin-format-date/src/flow-nodes.yml

```diff
@@ -11,15 +11,22 @@ flow-nodes:
         parameters:
           date:
             name: Date
-            description: The date to be formatted.
-            required: true
+            description: The date to be formatted example 2020/08/19. Disabling/not entering a date shows current date. See more valid date examples here https://momentjs.com/docs/#/parsing/string/
+            required: false
             initialType: string
             schema:
               type: string
           format:
             name: Format
-            description: The desired format. Supports all Moment.js supported formats.
-            required: true
+            description: The desired format. Supports all Moment.js formats for dates example YYYY-MM-DDTHH:mm:ssZ. See more valid format examples https://momentjs.com/docs/#/parsing/special-formats/
```

Comment:
I think we can properly describe the field a bit better - 

`Takes a string of tokens and replaces them with their corresponding values. For the full list of supported tokens and examples, see https://momentjs.com/docs/#/parsing/special-formats/`

Since we already have a separate prop describing default. - but if we still want to describe the default in a second way, then
`Takes a string of tokens and replaces them with their corresponding values. Defaults to ISO-8601 format. For the full list of supported tokens and examples, see https://momentjs.com/docs/#/parsing/special-formats/`